main.go

```diff
@@ -232,6 +232,15 @@ func (h k8sHelpers) GetNode(cli *client.Client) (*api.Node, error) {
 }
 
 func (h k8sHelpers) AddLabels(n *api.Node, labels Labels) {
+	// Remove nfd node labels that no longer apply
```

Comment:
Nit on the comment: We don't use `nfd` anywhere in the repo right now. Consider deleting `nfd` from the comment.